cypress/support/commands.js

```diff
@@ -1 +1,97 @@
+import AWS from 'aws-sdk';
 
+AWS.config = new AWS.Config({
+  accessKeyId: Cypress.env('AWS_ACCESS_KEY_ID'),
+  secretAccessKey: Cypress.env('AWS_SECRET_ACCESS_KEY'),
+  sessionToken: Cypress.env('AWS_SESSION_TOKEN'),
+  region: Cypress.env('AWS_REGION'),
+});
+
+const identityServiceProvider = new AWS.CognitoIdentityServiceProvider();
+
+const USER_POOL_ID = Cypress.env('AWS_USER_POOL_ID');
+const CLIENT_ID = Cypress.env('AWS_CLIENT_ID');
+
+const TEMP_PASSWORD = 'Abcdefghijkl%9';
+const NEW_PASSWORD = 'NewPassword%0';
```

Comment:
passord kunne kanskje blitt generert on-the-fly? type `uuid()` feks

Reply:
Har brukt `uuid` nå